tools/importers/SynchronizeTypographyTokens.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { ImportTypographyVariables, ImportTypographyTokens } from './SynchronizeTypographyTokens';
+const fs = require('fs');
+const tokensFilesConfig = require('./configs/FakeSynchronizeTypographyTokensConfig.json');
```

Comment:
instead of creating new file, can we define this valiable this file? This file could be confusing when you aren't aware it's only used in tests

Reply:
I'm not sure if I understand you correctly. I removed file and move it's content to variable in the spec file.